apps/prairielearn/package.json

```diff
@@ -112,7 +112,7 @@
     "fast-json-stable-stringify": "^2.1.0",
     "fetch-cookie": "^3.1.0",
     "file-type": "^21.0.0",
-    "filesize": "^10.1.6",
+    "filesize": "^11.0.2",
```

Comment:
This just bumped a lot of dependencies: https://github.com/avoidwork/filesize.js/blob/master/CHANGELOG.md